src/boards.h

```diff
@@ -38,6 +38,13 @@
 #ifndef BUTTON_FRESET
 #define BUTTON_FRESET   BUTTON_2
 #endif
+#ifndef BUTTON_DIR

```

Comment:
Thanks, but let's rename the `BUTTON_DIR` into `BUTTON_STATE_ACTIVE` 0 for active low, and 1 for active high, which is a little clearer.

We also don't need to define BUTTON_PULL, just check the BUTTON_STATE_ACTIVE and set up the opposite pullup in `button_init()` above